templates/zerver/register.html

```diff
@@ -53,7 +53,7 @@
 
         <div class="input-box full-width">
             <div class="help-box">
-                {{ _('Zulip has full unicode support, so you can spell your name exactly how you want it to appear.') }}
```

Comment:
Perhaps `Full name or [full name in some other language].` would be more indicative. And yep - planning to change the size / layout of the help messages soon :)